workflow-templates/1gp-ci.yml

```diff
@@ -24,102 +24,102 @@ on:
   workflow_call:
     inputs:
       config-path:
-        required: true
+        required: false
         type: string
 
 concurrency:
-    group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
-    cancel-in-progress: true
+  group: "${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}"
+  cancel-in-progress: ${{ github.ref != 'refs/heads/release' }}
 
 permissions:
   id-token: write # This is required for requesting the ID token for AWS actions
-  contents: read  # This is required for actions/checkout
+  contents: read # This is required for actions/checkout
 
 env:
-    BUILD_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-    COMMIT_SHA1: "${{ github.sha }}"
-    DD_AGENT_MAJOR_VERSION: "7"
-    DD_API_KEY: "${{ REPLACE_DD_API_SECRET }}"
-    DD_APP_KEY: "${{ REPLACE_DD_APP_SECRET }}"
-    DD_SITE: "datadoghq.com"
-    DEVOPS_SCRIPTS_PATH: "./devops-utility-belt/scripts"
-    DX_DELETE_SCRATCH_ORG: true
-    ENABLE_LINTING: "false"
-    GIT_BASE_BRANCH: "${{ github.base_ref }}"
-    GIT_BRANCH: "${{ github.ref_name }}"
-    GITHUB_TOKEN: "${{REPLACE_TOKEN_SECRET}}"
-    REPO_NAME: "${{ github.event.repository.name }}"
+  BUILD_URL: "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+  COMMIT_SHA1: "${{ github.sha }}"
+  DD_AGENT_MAJOR_VERSION: "7"
+  DD_API_KEY: "${{ REPLACE_DD_API_SECRET }}"
+  DD_APP_KEY: "${{ REPLACE_DD_APP_SECRET }}"
```

Comment:
Will these and other variables such as the `$GITHUB_TOKEN` not be the same between all projects? Should we just default them to something like `'${{ secrets.DD_API_KEY }}'`